### DISCUSSION TITLE
clean-up-deployment

### DISCUSSION THREAD

**Discussion by Celikoo** [Azure/avdaccelerator#442] [Q&A]
Hi,

the branch "clean-up-deployment" appears to be a work in progress.

Does somebody have created an un-/finished clean-up script?
I want to always be able to re-deploy a brownfield deployment to scale it more easily.
This should be working since, in my case, we only offer one generalized session host --> desktops.

I appreciate any input on this matter.

Best regards,
Celikoo

**Comment by danycontre** [comment]
Hey @Celikoo, clean-up script is still a work in progress.

What are you looking for is a way to redeploy a brownfield with different settings to update/add resources?
